dashboard/test/controllers/home_controller_test.rb

```diff
@@ -124,6 +124,22 @@ def setup_user_with_gallery
     assert_select 'a[href="/admin"]'
   end
 
+  test 'do not show levelbuilder links when not levelbuilder' do
+    sign_in create(:user)
+
+    get :index
+    assert_select 'a[href="/levels/new"]', 0
```

Comment:
Per offline discussion, for now, I'm going to leave the auth check where it is. It may get moved as part of a larger refactor and cleanup effort in #8671.